src/DevTools/Extension/components/Shell/Shell.tsx

```diff
@@ -1,74 +1,120 @@
-import React, { useRef } from 'react';
+import React, { useEffect, useRef } from 'react';
 import { Tabs } from '@mantine/core';
-import { useAtomValue } from 'jotai/react';
+import { useAtom } from 'jotai/react';
+import { flushSync } from 'react-dom';
+import Moveable from 'react-moveable';
 import { shellStylesAtom } from '../../../atoms/shell-styles';
 import { TabKeys, shellStyleDefaults } from '../../../constants';
 import { useDevtoolsJotaiStoreOptions } from '../../../internal-jotai-store';
 import { useSelectedShellTab } from './atoms';
 import { AtomViewer } from './components/AtomViewer';
 import { ErrorBoundary } from './components/ErrorBoundary';
 import { Header } from './components/Header';
-import { ShellResizeBar } from './components/ShellResizeBar';
 import { TabsHeader } from './components/TabsHeader';
 import { TimeTravel } from './components/TimeTravel';
 import { shellStyles } from './styles';
 
+function areWeTestingWithJest() {
+  return process.env.JEST_WORKER_ID !== undefined;
```

Comment:
Is this just a problem with jest? I wonder `react-movable`'s issue is limited to `jest` and not other testing libraries like `vitest` 🤔 

And what's the issue you're running into? I'm not a huge fan of having this logic in the library. Is there an alternate library that we could explore?

Reply:
This is related to jsdom

https://github.com/jsdom/jsdom/issues/1435

I just found out that if you just use `resize` you don't have a problem

Reply:
Got it! so the issue would likely persist for users mounting Jotai DevTools in the tests? Do we have any workarounds, maybe a polyfill or finding a different library?